### PR TITLE
First pass evaluation and status update: IA-2 control family for OCP, SA-5  for OCP and COS

### PR DESCRIPTION
**Pull request by thamill**
Updated status of:
  IA-2 - Previous status of 'not applicable', 
  IA-2(1) - Previous status 'partial', MFA access for network access to priv accounts is out of scope for OpenShift. New status 'not applicable'
  IA-2(2) - Previous status 'partial', MFA access for network access to non-priv accounts is out of scope for OpenShift. New status 'not applicable'
  IA-2(3) - Previous status 'partial', MFA access to priv accounts is out of scope for OpenShift. New status 'not applicable'
  IA-2(4) - Previous status 'partial', MFA access to local accounts is out of scope for OpenShift. New status 'not applicable'
  IA-2(5) - Previous status 'partial', individuals authenticated when a group authenticator is in place is out of scope for OpenShift. New status 'not applicable'
  IA-2(6) - Previous status 'partial', MFA authentication is an organization level configuration, out of scope for OCP. New status 'not applicable'
  IA-2(7) - Previous status 'partial', MFA authentication is an organization level configuration, out of scope for OCP. New status 'not applicable'
  IA-2(8) - Previous status 'complete', OCP natively supports modern TLS configuration, 3rd party IDM can be configured to use TLS. New status 'partial' as documentation for web and cli console is not updated for latest OCP release.
  IA-2(9) - Previous status 'complete', OCP natively supports modern TLS configuration, 3rd party IDM can be configured to use TLS. New status 'partial' as documentation for web and cli console is not updated for latest OCP release.
  IA-2(10) - Previous status 'complete', OCP does not manage SSO credentials, added auth doc link. New status 'not applicable'
  IA-2(11) - Unchanged, previous status 'not applicable'
  IA-2(12) - Previous status 'partial', OpenShift does not generate or manage PIV Federal credentials. New status 'not applicable'
  IA-2(13) - Unchanged, previous status 'not applicable'

Due to poor branch management, other edits have been rolled up into this PR. In the future, I will not do this.
Other controls included: SA-5 status change, including all not-withdrawn enhancements. All enhancements have been set to 'not applicable', text has been changed to reflect the org policy that the SA-5 controls should fall under.

### CODE REVIEW

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 1]
```diff
@@ -40,116 +44,159 @@
...
 
 - control_key: IA-2 (9)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: partial
   narrative:
     - text: |
+        OpenShift offers support for replay-resistant authentication
+        mechanisms, both for OpenShift console and 3rd party
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        mechanisms, both for OpenShift console and 3rd party
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 2]
```diff
@@ -40,116 +44,159 @@
...
 
 - control_key: IA-2 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        IA-2(2) is an organizational control out of scope for OpenShift configuration.
+        Multifactor authentication is not a service that OpenShift configuration
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Multifactor authentication is not a service that OpenShift configuration
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 3]
```diff
@@ -40,116 +44,159 @@
   implementation_status: planned
   narrative:
     - text: |
+        IA-2(1) is an organizational control out of scope for OpenShift configuration.
+        Multifactor authentication is not a service that OpenShift configuration
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Multifactor authentication is not a service that OpenShift configuration
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 4]
```diff
@@ -40,116 +44,159 @@
...
 
 - control_key: IA-2 (3)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        IA-2(3) is an organizational control out of scope for OpenShift configuration.
+        Multifactor authentication is not a service that OpenShift configuration
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Multifactor authentication is not a service that OpenShift configuration
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 5]
```diff
@@ -40,116 +44,159 @@
...
 
 - control_key: IA-2 (4)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        IA-2(4) is an organizational control out of scope for OpenShift configuration.
+        Multifactor authentication is not a service that OpenShift configuration
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Multifactor authentication is not a service that OpenShift configuration
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 6]
```diff
@@ -40,116 +44,159 @@
...
         https://issues.redhat.com/browse/CMP-303
 
 - control_key: IA-2 (5)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        IA-2(5) is an organizational control out of scope for OpenShift configuration.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        IA-2(5) is an organizational control out of scope for OpenShift configuration.
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 7]
```diff
@@ -40,116 +44,159 @@
...
         https://issues.redhat.com/browse/CMP-314
 
 - control_key: IA-2 (6)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        IA-2(6) is an organizational control out of scope for
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        IA-2(6) is an organizational control out of scope for
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 8]
```diff
@@ -40,116 +44,159 @@
...
         https://issues.redhat.com/browse/CMP-315
 
 - control_key: IA-2 (7)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        IA-2(7) is an organizational control out of scope for
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        IA-2(7) is an organizational control out of scope for
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 9]
```diff
@@ -40,116 +44,159 @@
...
 
 - control_key: IA-2 (8)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: partial
   narrative:
     - text: |
+        IA-(8)OpenShift offers support for replay-resistant authentication
+        mechanisms, both for OpenShift console and 3rd party
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        mechanisms, both for OpenShift console and 3rd party
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 10]
```diff
@@ -40,116 +44,159 @@
...
 - control_key: IA-2 (9)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: partial
   narrative:
     - text: |
+        OpenShift offers support for replay-resistant authentication
+        mechanisms, both for OpenShift console and 3rd party
+        authentication.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        authentication.
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 11]
```diff
@@ -40,116 +44,159 @@
...
 
 - control_key: IA-2 (10)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response for IA-2(10) is planned.
+        IA-2 (10) is an organizational control outside the scope of
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        IA-2 (10) is an organizational control outside the scope of
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 12]
```diff
@@ -40,116 +44,159 @@
...
 
 - control_key: IA-2 (11)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - text: |
-        Implementing a separate device from the system
-        gaining access to the information system using multifactor
+        IA-2 (11) is an organizational control outside the scope of
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        IA-2 (11) is an organizational control outside the scope of
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 13]
```diff
@@ -40,116 +44,159 @@
...
         guidance of OpenShift.
 
 - control_key: IA-2 (12)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        IA-2 (12) is an organizational control outside the scope of
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        IA-2 (12) is an organizational control outside the scope of
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 14]
```diff
@@ -177,6 +224,17 @@
   implementation_status: planned
   narrative:
     - text: |
+        OpenShift maintains an internal private network on the deployed
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        OpenShift maintains an internal private network on the deployed
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 15]
```diff
@@ -177,6 +224,17 @@
   implementation_status: planned
   narrative:
     - text: |
+        OpenShift maintains an internal private network on the deployed 
+        cluster that uniquely identifies internal nodes.
+        The satisfaction of this control is only for nodes within the
+        OpenShift private environment, and does not provide this
+        functionality to the organization environment.
+        doc: https://docs.openshift.com/container-platform/latest/networking/understanding-networking.html
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Further details can be found in the OpenShift documentation: 

        https://docs.openshift.com/container-platform/latest/networking/understanding-networking.html
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 16]
```diff
@@ -177,6 +224,17 @@
...
+        Creation of control response/guidance is being tracked in the
+        ComplianceAsCode backlog:
+
+        https://github.com/ComplianceAsCode/redhat/issues/964
+
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion

```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 17]
```diff
@@ -192,32 +250,50 @@
...
       text: |
+        a) Out of scope for OpenShift configuration, this control is
+        satisfied by organizational policies and procedures.
+        b) Out of scope for OpenShift configuration, this control is
+        satisfied by organizational policies and procedures.
+
         A control response is planned. Engineering progress can be tracked via:
 
         https://issues.redhat.com/browse/CMP-319
+
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 18]
```diff
@@ -192,32 +250,50 @@
...
         A control response is planned. Engineering progress can be tracked via:
 
         https://issues.redhat.com/browse/CMP-319
+        
     - key: b
       text: |
         A control response is planned. Engineering progress can be tracked via:
 
         https://issues.redhat.com/browse/CMP-319
+
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion

```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 19]
```diff
@@ -192,32 +250,50 @@
...
         https://issues.redhat.com/browse/CMP-319
+        
 - control_key: IA-3 (4)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        While OpenShift does create an internal private network within
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        While OpenShift does create an internal private network within
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 20]
```diff
@@ -192,32 +250,50 @@
...
+        
 - control_key: IA-3 (4)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        While OpenShift does create an internal private network within 
+        the cluster infrastructure, no devices are created that would
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        the cluster infrastructure, no devices are created that would
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 21]
```diff
@@ -192,32 +250,50 @@
...
 - control_key: IA-3 (4)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        While OpenShift does create an internal private network within 
+        the cluster infrastructure, no devices are created that would 
+        fall into the domain of this control. The cluster nodes that
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        fall into the domain of this control. The cluster nodes that
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 22]
```diff
@@ -192,32 +250,50 @@
...
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        While OpenShift does create an internal private network within 
+        the cluster infrastructure, no devices are created that would 
+        fall into the domain of this control. The cluster nodes that 
+        OpenShift is installed on, would require a non-OpenShift
+        organizational policy and procedure to satisfy this control.
+        doc: https://docs.openshift.com/container-platform/latest/networking/understanding-networking.html
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        
        Further details are provided in the OpenShift documentation:

        https://docs.openshift.com/container-platform/latest/networking/understanding-networking.html
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 23]
```diff
@@ -192,32 +250,50 @@
...
+
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion

```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 24]
```diff
@@ -192,32 +250,50 @@
...
+        While OpenShift does create an internal private network within 
+        the cluster infrastructure, no devices are created that would 
+        fall into the domain of this control. The cluster nodes that 
+        OpenShift is installed on, would require a non-OpenShift
+        organizational policy and procedure to satisfy this control.
+        doc: https://docs.openshift.com/container-platform/latest/networking/understanding-networking.html
+        
+        Creation of control response/guidance is being tracked in the
+        ComplianceAsCode backlog:
+
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion

```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 25]
```diff
@@ -258,33 +348,43 @@
   implementation_status: not applicable
   narrative:
     - text: |
-        This is an organizational procedural requirement
-        outside the scope of OpenShift configuration.
+        Receiving supervisor authorization to receive an
+        individual identifier is outside the scope of
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        individual identifier is outside the scope of
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 26]
```diff
@@ -258,33 +348,43 @@
...
+        of OpenShift configuration.
 
 - control_key: IA-4 (4)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - text: |
-        This is an organizational procedural requirement
+        Uniquely identifying each individual is an
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Uniquely identifying each individual is an
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 27]
```diff
@@ -258,33 +348,43 @@
...
 - control_key: IA-4 (5)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
+        Dynamic management of identifiers is not a responsibility
+        OpenShift is capable of fulfilling. This is an organiz-
+        ational procedural requirement outside the scope of
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        ational procedural requirement outside the scope of
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 28]
```diff
@@ -318,82 +418,144 @@
   narrative:
     - key: a
       text: |
-        This is an organizational procedural requirement
-        outside the scope of OpenShift configuration.
+        Verifying identity of individual receiveing the authenticator
+        is an organizational procedural requirement outside the
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        is an organizational procedural requirement outside the
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 29]
```diff
@@ -318,82 +418,144 @@
...
     - key: a
       text: |
-        This is an organizational procedural requirement
-        outside the scope of OpenShift configuration.
+        Verifying identity of individual receiveing the authenticator
+        is an organizational procedural requirement outside the 
+        scope of OpenShift configuration.
     - key: b
       text: |
+        Initial authenticator content is outside the scope of
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Initial authenticator content is outside the scope of
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 30]
```diff
@@ -318,82 +418,144 @@
...
       text: |
-        This is an organizational procedural requirement
-        outside the scope of OpenShift configuration.
+        Verifying identity of individual receiveing the authenticator
+        is an organizational procedural requirement outside the 
+        scope of OpenShift configuration.
     - key: b
       text: |
+        Initial authenticator content is outside the scope of 
+        OpenShift configuration.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        OpenShift configuration.
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 31]
```diff
@@ -318,82 +418,144 @@
...
+        scope of OpenShift configuration.
+        Creation of control response/guidance is being tracked in the
+        ComplianceAsCode backlog:
+
         A control response is planned. Engineering progress can be tracked via:
 
         https://issues.redhat.com/browse/CMP-325
     - key: g
       text: |
+        Refreshing and/or changing authenticators is outside the
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Refreshing and/or changing authenticators is outside the
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 32]
```diff
@@ -318,82 +418,144 @@
...
+        protected is outside the scope of OpenShift configuration.
+        Creation of control response/guidance is being tracked in the
+        ComplianceAsCode backlog:
+
         A control response is planned. Engineering progress can be tracked via:
 
         https://issues.redhat.com/browse/CMP-326
     - key: d
       text: |
+        Password aging and lifetime restriction enforcement is a
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Password aging and lifetime restriction enforcement is a
```

**Comment by shawndwells** [comment]
@stickler-ci lost its mind

**Comment by redhatrises** [comment]
This contains more than just AC-4... Please open a separate PR for each control.

**Pull request closed without merging**